### PR TITLE
Create a Subdomain

### PR DESCRIPTION
**Pull request by Marie59**
Opening the first PR for one of the 2 tutorials on subdomains.
This one is on how to set up a subdomain 

The ultimate goal is to create a learning pathway that helps people to discover the communities existing (the sig tuto), then if needed set up a subdomain, then how to create tools and finally how to add those tools to the subdomain (a 2nd tuto in a another PR)
An idea also was to add a tuto in the final Learning pathway on Galaxy Codex and on the GalaxyLabs (tuto needs to be don here)

### CODE REVIEW

File: topics/admin/tutorials/subdomain/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+---
+layout: tutorial_hands_on
+
+title: Create a subdomain for your community
```
Comment (hexylena) [inline topics/admin/tutorials/subdomain/tutorial.md:4]:
```suggestion
title: Create a subdomain for your community on UseGalaxy.eu
```

the instructions seem specific to EU so noting that here.

**Review by hexylena — COMMENTED** [review]
Looks fine technically, I leave it to @bgruening @mira-miracoli @sanjaysrikakulam to review the contents

File: topics/admin/tutorials/subdomain/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
+---
```
[the remark this answers is not in the document]
Comment (Marie59) [reply topics/admin/tutorials/subdomain/tutorial.md:1]: Thanks for the review !! Normally I updated the training according to your comments.

[the remark this answers is not in the document]
Comment (mira-miracoli) [reply topics/admin/tutorials/subdomain/tutorial.md:1]: Yes the process changed in between :)

**Review by mira-miracoli — APPROVED** [review]
Looks good, thank you!

**Comment by Marie59** [comment]
Hi all ! Is this tuto ready to ber merged then ? @hexylena @bgruening ?

**Review by bgruening — APPROVED** [review]

**Pull request merged**